app/Http/Controllers/PayloadController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PayloadCreateRequest;
+use App\Models\Condition;
+use App\Models\Payload;
+use App\Models\Webhook;
+use App\Repositories\Interfaces\PayloadRepositoryInterface as PayloadRepository;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class PayloadController extends Controller
+{
+    private $payloadRepository;
+
+    public function __construct(PayloadRepository $payloadRepository)
+    {
+        $this->payloadRepository = $payloadRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create($webhook_id)
+    {
+        // $webhooks = Webhook::pluck('id', 'name');
+        return view('payloads.create', compact('webhook_id'));
+    }
+
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 50</br>Variable "webhook_id" is not in valid camel caps format